tests/run-make/issue-88756-default-output/output-default.stdout

```diff
@@ -158,6 +158,9 @@ Options:
                         Comma separated list of types of output for rustdoc to
                         emit
         --no-run        Compile doctests without running them
+        --remap-path-prefix FROM=TO
+                        Remap source names in all output (compiler messages
+                        and output files)
```

Comment:
do we actually ever have paths in rustdoc output _files_?

Reply:
Yeah but I think this gives the wrong idea in that when I see these docs my first reaction is "wait we have paths in output files"?

(and we shouldn't do that)